styles.css

```diff
@@ -700,6 +700,7 @@ form textarea {
   .project-detail-article {
     height: 85vh;
     width: 50vw;
+    margin-bottom: 5%;
   }
```

Comment:
Great job team 🤝

- [ ] Just like the previous reviewer pointed out, kindly fix the `margin-bottom` of your mobile popup so that your design can replicate the design provided in the Figma template.
![mobile screen](https://user-images.githubusercontent.com/63917725/189346491-c8b0be3d-75eb-4d7d-a854-f1f27ccfc1de.PNG)